frontend/src/model/beta/overview/TemplatePage.tsx

```diff
@@ -12,69 +15,79 @@ type TemplatePageProps = {
 export default function TemplatePage({ model }: TemplatePageProps) {
   const theme = useTheme()
   const router = useRouter()
+  const { schemas, isSchemasLoading, isSchemasError } = useGetSchemas(model.card.schemaId)
```

Comment:
You're grabbing the `schemas` for this page, but then don't seem to be using them anywhere?